docs/store-operations/shipping/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,75 @@
+# Shipping - International Enhancements for Multi-Storefront
+
+When you sell internationally or to mulitple customer segments, you may want the available shipping methods for a shipping zone to depend on the shopper's region or segment.
+
+To localize your storefront, you can have zones that display different methods depending on the storefront channel. 
+Specify the storefront channel(s) where you want a shipping method to appear when you create a method for a zone. Shoppers only see the shipping method for the zone if it applies to a channel.
```

Comment:
Specify the storefront channel(s) where you want a shipping method to appear when you create a method for a zone.
reads a little strange to me.
What about 
Specify which storefront channel(s) you want a shipping method to appear on when you create a method for a zone.

and 
Shoppers will only see the shipping method for the zone if it applies to a channel.

Reply:
Thanks for catching that. I also reworded slightly because I realized my original sentence sounded like the shipping method was going to appear when you are creating a method.